nx/include/switch/services/btm.h

```diff
@@ -79,10 +88,19 @@ Result btmSetWlanMode(BtmWlanMode mode);
 Result btmAcquireDeviceInfoEvent(Event* out_event);
 
 /**
- * @brief GetDeviceInfo
+ * @brief GetDeviceInfo [1.0.0-12.1.0]
  * @param[out] out \ref BtmDeviceInfoList
  */
-Result btmGetDeviceInfo(BtmDeviceInfoList *out);
+Result btmLegacyGetDeviceInfo(BtmDeviceInfoList *out);
+
+/**
+ * @brief GetDeviceInfo [13.0.0+]
+ * @param[in] id Id
+ * @param[out] out \ref BtmDeviceInfo
+ * @param[in] count Size of the out array in entries.
+ * @param[out] total_out Total output entries.
+ */
+Result btmGetDeviceInfo(u32 id, BtmDeviceInfo *out, size_t count, s32 *total_out);
```

Comment:
btm_types.h changes missing for the updated BtmDeviceInfo.

Reply:
Good catch, I didn't realise it had changed